pkg/tools/multiexecutor/multi_executor_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package multiexecutor_test
+
+import (
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/goleak"
+
+	"github.com/networkservicemesh/sdk/pkg/tools/multiexecutor"
+)
+
+const (
+	parallelCount = 1000
+)
+
+var ids = []string{
+	"0", "1", "2", "3", "4", "5", "6", "7", "8", "9",
+}
```

Comment:
I feel you could generate it in the test body to prevent static data invalidating between a couple of tests running.

Reply:
Removed.